features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/edit/RoomDetailsEditPresenter.kt

```diff
@@ -65,7 +65,7 @@ class RoomDetailsEditPresenter @Inject constructor(
         // just erase the local value when the room field has changed
         var roomAvatarUri by rememberSaveable(room.avatarUrl) { mutableStateOf(room.avatarUrl?.toUri()) }
 
-        var roomName by rememberSaveable { mutableStateOf((room.name ?: room.displayName).trim()) }
+        var roomName by rememberSaveable { mutableStateOf(room.displayName.trim()) }
```

Comment:
We need to use the new field `RoomInfo.rawName` here, but this can be handled separately.

It will fix the form per-filled with a computed room name like "Alice, Bob and 3 others".

Reply:
Using `RoomInfo.rawName` needs some other changes so I think it's probably better to do it on a separate PR.